src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -10,24 +10,52 @@
     {{ parent() }}
 {% endblock %}
 
-{% block scripts %}
-    <script src="{{ asset('bundles/packagistweb/js/view.js')}}"></script>
-
+{% block head_additions %}
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/monokai_sublime.min.css">
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/highlight.min.js"></script>
-    <script>
-      $(document).ready(function() {
-        hljs.configure({
-          tabReplace: 4,
-          languages: ['apache', 'bash', 'ini', 'javascript', 'json', 'nginx', 'php', 'sql', 'css', 'html', 'twig', 'xml', 'less', 'scss', 'yaml']
-        });
-        $('div.readme pre, div.readme code').each(function(i, block) {
-          hljs.highlightBlock(block);
+    <script type="text/javascript">
+      WebFontConfig = {
+        google: { families: [ 'Source+Code+Pro::latin' ] }
+      };
+      (function() {
+        var wf = document.createElement('script');
+        wf.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
```

Comment:
Maybe better to use `//` here instead of `https://`. That way it will use the protocol that the user is currently using to access Packagist, e.g. http or https (if I am not mistaken).